lightstep/recorder.py

```diff
@@ -58,7 +58,8 @@ def record_span(self,span):
                 else:
                     event = log.event
             logs.append(ttypes.LogRecord(stable_name= event, payload_json= log.payload))
-        logging.warn('Reporting span %s \n with logs %s', _pretty_span(span), _pretty_logs(logs))
```

Comment:
@shayel the intention with `LoggingRecorder` is to, well, log everything (per the comment at line 46).

If you don't want to record data in LightStep and also don't want any debugging information, perhaps the best thing would be to just use the default [noop OpenTracing implementation](https://github.com/opentracing/opentracing-python/blob/master/opentracing/__init__.py#L33). Does that make sense?

We could do `isEnabledFor(logging.INFO)` + `logging.info()` instead of `warn`) regardless.